dandiapi/api/services/asset/__init__.py

```diff
@@ -168,15 +168,16 @@ def add_asset_to_version(
         raise ZarrArchiveBelongsToDifferentDandisetError
 
     with transaction.atomic():
-        # Creating an asset in an OPEN dandiset that points to an embargoed blob results in that
-        # blob being unembargoed
+        # Creating an asset in an OPEN dandiset that points to an
+        # embargoed blob results in that blob being unembargoed.
+        # NOTE: This only applies to asset blobs, as zarrs cannot belong to
+        # multiple dandisets at once.
         if (
-            asset_blob is not None
+            asset_blob
```

Comment:
What is the reason for removing the `is not None` condition here?

Reply:
Good point, i'll revert that.